### DISCUSSION TITLE
502 Bad getaway error in termux

### DISCUSSION THREAD

**Discussion by UBV76** [termux/termux-app#3077] [General]
`502  Bad Gateway
Reading package lists... Done
E: The repository 'https://dl.bintray.com/grimler/game-packages-24 games Release' does not have a Release file.
N: Metadata integrity can't be verified, repository is disabled now.
N: Possible cause: repository is under maintenance or down (wrong sources.list URL?).
E: The repository 'https://dl.bintray.com/grimler/science-packages-24 science Release' does not have a Release file.
N: Metadata integrity can't be verified, repository is disabled now.
N: Possible cause: repository is under maintenance or down (wrong sources.list URL?).````
What should I do?

**Comment by agnostic-apollo** [comment]
https://github.com/termux/termux-app#Installation

https://github.com/termux/termux-app#google-play-store-deprecated
